### DISCUSSION TITLE
Crazyflie m2 led light remains solid blue while working on crazyswarm

### DISCUSSION THREAD

**Discussion by rajatag77** [USC-ACTLab/crazyswarm#669] [Q&A] [answered]
Hi, 
I was working on Crazyswarm. It worked for a few runs then this problem occurred.
When I turn on my crazyflie only M2 lights up solid. If I try to put it on bootloader mode, only m2 blinks.
Only the M2 Led is fully lit (Blue) when I plug the USB cable.
With battery plug-in, same result: only M2 blue Led fully lit.
I can't charge the battery.
I have several batteries (crazyFlie2 batteries) and tested with different ones: the same result.
I also tested several USB cables, without success.
Board is clean. No big crash or shock on the board...

I was working on Crazyswarm. It worked for a few runs then this problem occurred.
VCC measured is 3V.
Also, I have started the board in bootloader mode by holding the power button and starting it for 3 seconds until the M2 led starts to blink to check if M3 led starts to blink then. No luck...

The VCC is at 2.99V so that's not the problem.
I tried using the cfclient to reset the bootloader using the instructions (replug radio, etc) and nothing happens.
I have changed out the batteries and left the crazyflie unplugged for a few minutes.
I tried powering it just over USB and there is no change.
I tried using the cfloader flash tool but it says it can not connect to the bootloader.
I also tried cold bootloader mode with USB as well as Batteries.

**Comment by whoenig** [comment, marked as answer]
This seems like a hardware/firmware issue, and not a Crazyswarm issue.

Duplicate of https://github.com/bitcraze/crazyflie-firmware/issues/1090.

**Reply by knmcguire** [reply]
We will answer this in another duplicate in our discussions: https://github.com/orgs/bitcraze/discussions/172
